### DISCUSSION TITLE
Unable to Login to Capawesome CLI using Correct Credentials

### DISCUSSION THREAD

**Discussion by HKornchanok** [capawesome-team/cli#9] [Q&A] [answered]
I am encountering an issue while trying to log in to the Capawesome CLI using the npx capawesome login --token command. Even though I provide the same credentials that work on the web, I receive an "Invalid email or password" error. Below is the output from the command:

```
kornchanok@Kornchanoks-Laptop % npx capawesome login

✔ Enter your email: kornchanok@****.com
◇ Enter your password: ▪▪▪▪▪▪▪▪▪▪
◐ Logging in...

ERROR Invalid email or password.
```

I have double-checked that the credentials are correct and have successfully logged into the web interface with the same email and password. However, it still fails on the CLI.

Any help or guidance on resolving this issue would be appreciated!

**Comment by robingenz** [comment, marked as answer]
Hi @HKornchanok, please provide the following information:

1. Which Capawesome CLI version do you use? Please run `npx capawesome --help` and paste the output here.
2. Which sign-in provider did you choose for Capawesome Cloud (email, github, gitlab)?

What should definitely work is the following:

1. Make sure you are using the latest version of the Capawesome CLI: `npm i -g @capawesome/cli@latest`
2. Create a token on the [Tokens](https://cloud.capawesome.io/settings/tokens) page.
3. Run the following command and replace `<TOKEN>` with the created token: `npx capawesome login --token <TOKEN>`

You should now be logged in.
